python_modules/libraries/dagster-sigma/dagster_sigma/translator.py

```diff
@@ -117,6 +118,16 @@ def get_asset_spec(self, data: Union[SigmaDataset, SigmaWorkbook]) -> AssetSpec:
                 ),
                 "dagster_sigma/properties": MetadataValue.json(data.properties),
                 "dagster_sigma/lineage": MetadataValue.json(data.lineage),
+                **(
+                    {
+                        "dagster_sigma/materialization_schedules": MetadataValue.json(
```

Comment:
Could we start using a custom metadata set for Sigma? Like [here](https://github.com/dagster-io/dagster/blob/76c18ddb07543d481f702cd33bc97d3878e1e7ae/python_modules/libraries/dagster-airbyte/dagster_airbyte/translator.py#L149-L156) for Airbyte?

Reply:
Updated